CMakeLists.txt

```diff
@@ -25,31 +25,31 @@ ecbuild_bundle_initialize()
 if(DEFINED ENV{jedi_cmake_ROOT})
   include( $ENV{jedi_cmake_ROOT}/share/jedicmake/Functions/git_functions.cmake )
 else()
-  ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/jcsda/jedi-cmake.git" TAG 1.3.0 )
+  ecbuild_bundle( PROJECT jedicmake GIT "https://github.com/jcsda/jedi-cmake.git" TAG 1.4.0 RECURSIVE )
   include( jedicmake/cmake/Functions/git_functions.cmake )
 endif()
 
 #ecbuild_bundle( PROJECT eckit    GIT "https://github.com/ecmwf/eckit.git" TAG 1.19.0 )
 #ecbuild_bundle( PROJECT fckit    GIT "https://github.com/ecmwf/fckit.git" TAG 0.9.5 )
-#ecbuild_bundle( PROJECT atlas    GIT "https://github.com/ecmwf/atlas.git" TAG 0.29.0 )
+#ecbuild_bundle( PROJECT atlas    GIT "https://github.com/ecmwf/atlas.git" TAG 0.30.0 )
 
-ecbuild_bundle( PROJECT oops     GIT "https://github.com/jcsda/oops.git"        TAG 1.3.0 )
-ecbuild_bundle( PROJECT vader    GIT "https://github.com/jcsda/vader.git"       TAG 1.0.0 )
-ecbuild_bundle( PROJECT saber    GIT "https://github.com/jcsda/saber.git"       TAG 1.3.0 )
+ecbuild_bundle( PROJECT oops     GIT "https://github.com/jcsda/oops.git"        TAG 1.4.0 )
+ecbuild_bundle( PROJECT vader    GIT "https://github.com/jcsda/vader.git"       TAG 1.1.0 )
+ecbuild_bundle( PROJECT saber    GIT "https://github.com/jcsda/saber.git"       TAG 1.4.0 )
 
-ecbuild_bundle( PROJECT crtm     GIT "https://github.com/jcsda/crtm.git"        TAG v2.3-jedi.1 )
-ecbuild_bundle( PROJECT gsw      GIT "https://github.com/jcsda/GSW-Fortran.git" TAG v3.06 )
-ecbuild_bundle( PROJECT ioda     GIT "https://github.com/jcsda/ioda.git"        TAG 2.2.0 )
-ecbuild_bundle( PROJECT ufo      GIT "https://github.com/jcsda/ufo.git"         TAG 1.3.0 )
+ecbuild_bundle( PROJECT crtm     GIT "https://github.com/jcsda/crtm.git"        TAG v2.4-jedi )
```

Comment:
In release/skylab-v2 this line (CRTM) is commented out. Is that supposed to be the same here?

Reply:
Yes! Good catch, thanks

Reply:
Fixed in https://github.com/JCSDA/jedi-bundle/pull/1/commits/f365b6be4d8cb840aace43cc2d9b10bcb46950f7